### PR TITLE
[prepare-track] Test docker agent and add logs to install_with

### PR DESCRIPTION
**Pull request by pmusa**
- adds logs to install_with_*
- adds docker case to test_agent
- runs the final install command in the background

### CODE REVIEW

File: common/prepare-track/init.sh [hunk 1]
```diff
@@ -325,7 +335,18 @@ function test_agent () {
...
-        FOUND_COLLECTOR=`kubectl logs -l app.kubernetes.io/instance=sysdig-agent -n sysdig-agent --tail=-1 2> /dev/null | grep "collector:" | head -n1 | awk '{print $NF}'`
+        case "$INSTALL_WITH" in
+            helm)
+                FOUND_COLLECTOR=`kubectl logs -l app.kubernetes.io/instance=sysdig-agent -n sysdig-agent --tail=-1 2> /dev/null | grep "collector:" | head -n1 | awk '{print $NF}'`
+                ;;
+            docker)
+                FOUND_COLLECTOR=`docker logs sysdig-agent 2>&1 | grep "collector:" | head -n1 | awk '{print $NF}'`
+                ;;
+            host)
+                FOUND_COLLECTOR=`echo 'journalctl -u dragent | grep -q "${CONNECTED_MSG}"'`
```
Comment (pabloopez) [inline common/prepare-track/init.sh:346]: I think this should be:
```suggestion
                FOUND_COLLECTOR=`echo 'journalctl -u dragent | grep "collector:" | head -n1 | awk '{print $NF}''`
```

File: common/prepare-track/install_with_docker.sh [hunk 2]
```diff
@@ -6,6 +6,7 @@
 #   install_with_docker.sh ${CLUSTER_NAME} ${ACCESS_KEY} ${COLLECTOR}
 ##
 
+OUTPUT=/opt/sysdig/docker_install.out
```
Comment (pabloopez) [inline common/prepare-track/install_with_docker.sh:9]: this is a nice add, it will make debugging easier.

**Review by pabloopez — CHANGES_REQUESTED** [review]

**Comment by pabloopez** [comment]
Following the discussion with @pmusa in Slack, the suggestion above has no effect right now as right now it is not reading the agent logs when installed directly in the host. I suggest (not tested) to check in `/opt/draios/logs/draios.log`. This should not block this changes merged, all looks good.

File: common/prepare-track/init.sh (hunk 1 above)
Comment (pmusa) [reply common/prepare-track/init.sh:346]:
```suggestion
                FOUND_COLLECTOR=`grep -q "collector:" /opt/draios/logs/draios.log | head -n1 | awk '{print $NF}'`
```

File: common/prepare-track/init.sh [hunk 3]
```diff
@@ -312,7 +312,17 @@ function test_agent () {
...
-        kubectl logs -l app.kubernetes.io/instance=sysdig-agent -n sysdig-agent --tail=-1 2> /dev/null | grep "${CONNECTED_MSG}" &> /dev/null
+        case "$INSTALL_WITH" in
+            helm)
+                kubectl logs -l app.kubernetes.io/instance=sysdig-agent -n sysdig-agent --tail=-1 2> /dev/null | grep -q "${CONNECTED_MSG}"
+                ;;
+            docker)
+                docker logs sysdig-agent 2>&1 | grep -q "${CONNECTED_MSG}"
+                ;;
+            host)
+                echo 'journalctl -u dragent | grep -q "${CONNECTED_MSG}"'
```
Comment (pmusa) [inline common/prepare-track/init.sh:323]:
```suggestion
                grep -q "${CONNECTED_MSG}" /opt/draios/logs/draios.log
```

**Comment by pmusa** [comment]
I modified the code to use the log file. Can you do a final check @pabloopez ?

File: common/prepare-track/init.sh [hunk 4]
```diff
@@ -325,7 +335,18 @@ function test_agent () {
...
-        FOUND_COLLECTOR=`kubectl logs -l app.kubernetes.io/instance=sysdig-agent -n sysdig-agent --tail=-1 2> /dev/null | grep "collector:" | head -n1 | awk '{print $NF}'`
+        case "$INSTALL_WITH" in
+            helm)
+                FOUND_COLLECTOR=`kubectl logs -l app.kubernetes.io/instance=sysdig-agent -n sysdig-agent --tail=-1 2> /dev/null | grep "collector:" | head -n1 | awk '{print $NF}'`
+                ;;
+            docker)
+                FOUND_COLLECTOR=`docker logs sysdig-agent 2>&1 | grep "collector:" | head -n1 | awk '{print $NF}'`
+                ;;
+            host)
+                FOUND_COLLECTOR=`grep -q "collector:" /opt/draios/logs/draios.log | head -n1 | awk '{print $NF}'`
```
Comment (pmusa) [inline common/prepare-track/init.sh:346]:
```suggestion
                FOUND_COLLECTOR=`grep "collector:" /opt/draios/logs/draios.log | head -n1 | awk '{print $NF}'`
```

**Comment by pabloopez** [comment]
Sure, I self-requested a review @pmusa.

**Review by pabloopez — APPROVED** [review]
LGTM, I see you included suggested changes. Thank you.

**Review by pabloopez — APPROVED** [review]
there you go

**Pull request merged**